web/controllers/organization_membership_controller.ex

```diff
@@ -19,19 +21,12 @@ defmodule CodeCorps.OrganizationMembershipController do
     render(conn, "index.json-api", data: memberships)
   end
 
-  def show(conn, %{"id" => id}) do
-    membership =
-      OrganizationMembership
-      |> preload([:organization, :member])
-      |> Repo.get!(id)
-
-    render(conn, "show.json-api", data: membership)
+  def show(conn, %{"id" => _id}) do
+    render(conn, "show.json-api", data: conn.assigns.organization_membership)
```

Comment:
Explained it in the other PR. 

Both `load_and_authorize_resource` and `load_resource` plugs provided by `Canary` load the endpoint resource using (by default) the `id` params key. If the plugs manage to load it, it's assigned to `conn.assigns.underscored_model_name`. If it fails, the `handle_not_found` handler, which we defined, is triggered. This handler halts and renders a 404 JSON error.

When we do it manually, instead, the loading just fails and we end up with a 404 HTML error, which is a bug on it's own. 

Also, since we do use `load_and_authorize_resource` either way, that means the resource already gets loaded once. When we proceed to load it manually, we're actually end up loading it twice for endpoints that have authorization.